### PR TITLE
Add the requirements in readme

### PR DESCRIPTION
**Pull request by testforstephen**
fix https://github.com/Microsoft/vscode-java-debug/issues/107 and https://github.com/Microsoft/vscode-java-debug/issues/126

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ A lightweight Java Debugger based on [Java Debug Server](https://github.com/Micr
 - Threads
 - Debug console
 
+## Requirements
+- JDK (version 1.8.0 and later)
```
Comment (yaohaizh) [inline README.md]: and ==> or

File: README.md [hunk 2]
```diff
@@ -16,6 +16,11 @@ A lightweight Java Debugger based on [Java Debug Server](https://github.com/Micr
...
+- VS Code (version 1.17.0 and later)
```
Comment (yaohaizh) [inline README.md]: and or

File: README.md [hunk 3]
```diff
@@ -70,4 +75,4 @@ You can find the full list of issues at [Issue Tracker](https://github.com/Micro
 This extension is licensed under [MIT License](https://github.com/Microsoft/vscode-arduino/blob/master/LICENSE.txt).
 
 ## Data/Telemetry
-Java Debug Extension for Visual Studio Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](http://go.microsoft.com/fwlink/?LinkId=521839) to learn more. 
\ No newline at end of file
+Java Debug Extension for Visual Studio Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](http://go.microsoft.com/fwlink/?LinkId=521839) to learn more.
```
Comment (yaohaizh) [inline README.md]: Debugger for Java...

Comment (testforstephen) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (testforstephen) [reply README.md]: fixed.

File: README.md (hunk 1 above)
Comment (testforstephen) [reply README.md]: fixed.

**Pull request merged**